lib/kuzzle/kuzzle.js

```diff
@@ -197,6 +199,10 @@ class Kuzzle extends KuzzleEventEmitter {
 
       this.log.info('[✔] Core components loaded');
 
+      const executedCount = await this.install(options.installations);
+
+      this.log.info(`[✔] ${executedCount} new installations executed`);
+
```

Comment:
To me, having a log saying `"N new installations executed"` makes no sense :facepalm:

Reply:
Since we don't really know how to name those installations / setups / deployements(:trollface:), we may let the developer with whatever names he has chosen and print a feedback for each of those (which should make @xbill82 happy), I see something like that:
```javascript
this.log.info('[✔] ${installation.id}: Successfully executed');
this.log.info('[✔] ${installation.id}: Already executed on ${retreivedInstallation.installedAt}');
```
```sh-session
[✔] Mappings 1.8.0: Already executed on Tue Aug 19 2020 23:15:30 GMT+0100 (Central European Standard Time)
[✔] Mappings 2.4.3: Successfully executed
```

Reply:
I see your point, Mathieu. Bootstrap is not the word: I understand the "only once" character of the thing. I' out of ideas for this feature: but installation looks ugly. Did you already consider "Setup"? Even if it's not the word either...

Reply:
"Setup" may be prettier but since we've chosen "install" for the name of the function I feel like it's less consistent... Do you think they'll understand the link between install and setup when working on the backend?

Reply:
I have to admit I have no more ideas on this. Install feels ugly, but seems to be the only word that fits.

Reply:
I agree with @Leodau , I don't see the point of that log message.

I would rather have a log explaining what new installs have been run (with their ids), and nothing otherwise.

Also, since we went for  `app.install`, we might as well go all the way with that name, should make things easier when users are trying to understand log messages.

Reply:
Go for it, I agree with that :+1: